__tests__/app.test.js

```diff
@@ -33,3 +33,15 @@ describe('/api/some-random-api', () => {
         })
     })
 })
+
+describe('/api', () => {
+    test('GET 200: responds with an object describing all the available endpoints on the API', () => {
+        const endpoints = require('../endpoints.json')
```

Comment:
You can move this to the top of the file with the other things you've required in